configurator/rootfs/etc/services.d/configurator/run

```diff
@@ -4,9 +4,12 @@
 # ==============================================================================
 export HC_DIRSFIRST
 export HC_ENFORCE_BASEPATH
+export HC_GIT
 export HC_HASS_API_PASSWORD
 export HC_IGNORE_PATTERN
 
+bashio::log.info "Starting Configurator run script"
+
```

Comment:
This is already outputted by S6 and not needed.

```suggestion
```

Reply:
I'm using a bit of unique setup. I have my Raspberry Pi config folder setup with remote git repository which reside on my PC. This is achieved with separate-git-dir git option. Problem is that configurator doesn't handle this too well and when initialized I get the following error:
WARNING:2020-11-06 20:28:05,602:hass_configurator.configurator:Cmd('git') failed due to: exit code(128)
  cmdline: git status --porcelain --untracked-files
  stderr: 'fatal: not a git repository: /config/C:\ttt\HomeAssistant\config\.git

This fix allows me to use configurator (which is really handy) even when it doesn't support my git setup.

BTW: The added tracing just helped a novice like to find out what is running when but I dont have any problem with removing it.